src/index.js

```diff
@@ -3,6 +3,8 @@ import loaderUtils from 'loader-utils';
 import validateOptions from 'schema-utils';
 import schema from './options.json';
 
+/* eslint-disable multiline-ternary */
```

Comment:
Move this to the top please

Reply:
@michael-ciniawsky before `import`?

Reply:
Yep

Reply:
@michael-ciniawsky done :+1:

Reply:
@michael-ciniawsky after `options.publicPath || ''` we don't need this comment :+1: